src/SamplesApp/UITests.Shared/Windows_UI_Xaml_Media_Animation/SetTargetProperty.xaml.cs

```diff
@@ -0,0 +1,127 @@
+﻿using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Runtime.InteropServices.WindowsRuntime;
+using Uno.UI.Samples.Controls;
+using Windows.Foundation;
+using Windows.Foundation.Collections;
+using Windows.UI.Xaml;
+using Windows.UI.Xaml.Controls;
+using Windows.UI.Xaml.Controls.Primitives;
+using Windows.UI.Xaml.Data;
+using Windows.UI.Xaml.Input;
+using Windows.UI.Xaml.Media;
+using Windows.UI.Xaml.Media.Animation;
+using Windows.UI.Xaml.Navigation;
+
+// The Blank Page item template is documented at https://go.microsoft.com/fwlink/?LinkId=234238
+
+namespace UITests.Shared.Windows_UI_Xaml_Media_Animation
+{
+    [Sample("Animations")]
+    public sealed partial class SetTargetProperty : Page
+    {
+		private static readonly TimeSpan AnimationDuration = TimeSpan.FromMilliseconds(1000);
```

Comment:
Might want to reduce the duration here, or raise the wait time above:
> ```cs
> await Task.Delay(1000); // Wait for animation.
> ```
We don't want the test to fail due to lag. Not saying that it will, but it is a possibility?

Reply:
Indeed it is best to ensure that there's a condition to observe, rather than a delay. We can probably hook onto the storyboard events for that.